packages/url_launcher/url_launcher/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: url_launcher
 description: Flutter plugin for launching a URL on Android and iOS. Supports
   web, phone, SMS, and email schemes.
 author: Flutter Team <flutter-dev@googlegroups.com>
-homepage: https://github.com/flutter/plugins/tree/master/packages/url_launcher
+homepage: https://github.com/flutter/plugins/tree/master/packages/url_launcher/url_launcher
 version: 5.1.4
```

Comment:
we should bump the minor version so we can publish the updated link to pub

Reply:
sorry - the patch version

Reply:
Done